toolkit/imageconfigs/core-legacy-unattended-hyperv.json

```diff
@@ -0,0 +1,67 @@
+{
+    "Disks": [
+        {
+            "PartitionTableType": "gpt",
+            "TargetDisk": {
+                "Type": "path",
+                "Value": "/dev/sda"
+            },
+            "Artifacts": [
```

Comment:
Do we need to keep the Artifacts field in this image configuration?

Reply:
Good catch. It's not needed